cmd/integrationArtifactUpdateConfiguration.go

```diff
@@ -81,7 +81,7 @@ func runIntegrationArtifactUpdateConfiguration(config *integrationArtifactUpdate
 	jsonBody, jsonErr := json.Marshal(jsonObj)
 
 	if jsonErr != nil {
-		return errors.Wrap(jsonErr, "input json body is invalid")
+		return errors.Wrapf(jsonErr, "input json body is invalid %v", config.ParameterValue)
```

Comment:
Rather something like `"input json body is invalid for parameterValue %q", config.ParameterValue`